src/components/OnboardingScreen.js

```diff
@@ -150,6 +162,7 @@ export function OnboardingScreen({
               </div>
             </div>
             <OnboardingList notify={notify} />
+            {(NewfoldRuntime.isAdmin &&  <FacebookConnectButton />) || ''}
```

Comment:
I assume we're doing away with this isAdmin check since that isn't being added to the runtime.